### PR TITLE
sc-60: Sc 60/create nft page 2

### PR DESCRIPTION
**Pull request by akshay-rakheja**
Minter works! -> had to deploy the contract on ropsten. i think we were just doing it locally until now within hardhat?
Can make a new ticket for these:
Need to refactor to combine some helper functions, need to use styling . 
Until then plzzz approve.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 .env
 .eslintcache
 /cache
+package-lock.json
```
Comment (Justin1002) [inline .gitignore:6]: you shouldn't git ignore dis :3

File: frontend/.gitignore [hunk 2]
```diff
@@ -19,6 +19,8 @@
 .env.test.local
 .env.production.local
 
+package-lock.json
```
Comment (Justin1002) [inline frontend/.gitignore:22]: you shouldn't git ignore dis :3

File: frontend/package.json [hunk 3]
```diff
@@ -11,13 +12,16 @@
...
     "react": "^17.0.2",
+    "react-app-polyfill": "^3.0.0",
     "react-dom": "^17.0.2",
-    "react-multi-carousel": "^2.6.5",
     "react-dotenv": "^0.1.3",
+    "react-multi-carousel": "^2.6.5",
     "react-router-dom": "^6.2.1",
-    "react-scripts": "5.0.0"
+    "react-scripts": "4.0.3",
+    "eslint": "^7.11.0"
```
Comment (Justin1002) [inline frontend/package.json:24]: why did you add eslint as a dependency? we run it from the root folder and it should also check frontend

File: frontend/src/components/Minters.jsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  const onChange = e =>{
+    var reader = new FileReader();
+    var file = e.target.files[0];
+    reader.onload = function(upload) {
+        setFile({
+            image: upload.target.result
+        });
+    }
+    reader.readAsDataURL(file); 
+    console.log(file);
```
Comment (Justin1002) [inline frontend/src/components/Minters.jsx:78]: do you still need this console log?

File: frontend/src/hooks/use-interact.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+import { nftaddress } from '../config';
+
+require('dotenv').config();
+const alchemyKey = process.env.REACT_APP_ALCHEMY_KEY;
+const contractAddress = nftaddress;
+const contractABI = require('../artifacts/contracts/NFT.sol/NFT.json');
+const { createAlchemyWeb3 } = require('@alch/alchemy-web3');
+const web3 = createAlchemyWeb3(alchemyKey);
+
+export const connectWallet = async () => {
```
Comment (Justin1002) [inline frontend/src/hooks/use-interact.js:11]: I thought you were gonna use the useWalletConnection functions

File: .gitignore (hunk 1 above)
Comment (akshay-rakheja) [reply .gitignore:6]: Was told it's probably a good idea. Was also facing some massive merge conflicts. We can just npm i for the latest and greatest.

File: frontend/package.json (hunk 3 above)
Comment (akshay-rakheja) [reply frontend/package.json:24]: Had some errors initially because of this.

File: frontend/src/components/Minters.jsx (hunk 4 above)
Comment (akshay-rakheja) [reply frontend/src/components/Minters.jsx:78]: Nope not really. I will be refactoring this entire prt in another ticket to make use of use wallet connection and others.

File: frontend/src/hooks/use-interact.js (hunk 5 above)
Comment (akshay-rakheja) [reply frontend/src/hooks/use-interact.js:11]: Yup leaving that for another ticket. Don't wanna mess up too much with stuff that's not broken yet.

**Review by connie-ho — APPROVED** [review]
